rosbag2_compression/src/rosbag2_compression/zstd_compressor.cpp

```diff
@@ -26,6 +26,15 @@
 
 namespace rosbag2_compression
 {
+ZstdCompressor::ZstdCompressor()
+{
+  zstd_context_ = ZSTD_createCCtx();
```

Comment:
I didn't know about this before but dumping here for reference (might be good as a comment too :) )

[From the manual](https://facebook.github.io/zstd/zstd_manual.html):

> When compressing many times, it is recommended to allocate a context just once, and re-use it for each successive compression operation.
> This will make workload friendlier for system's memory.
> Note : re-using context is just a speed / resource optimization. It doesn't change the compression ratio, which remains identical.
> Note 2 : In multi-threaded environments, use one different context per thread for parallel execution.